master/custom/builders.py

```diff
@@ -89,6 +91,7 @@
 
     # macOS x86-64 clang
     ("x86-64 macOS", "billenstein-macos", UnixBuild),
+    ("x86-64 MacOS Intel NoGIL", "itamaro-macos-intel-aws", UnixNoGilBuild),
```

Comment:
If free-threaded CIs are triggered anyway, Can we update this buildbot as refleak checker?

Reply:
afaik the buildbot test suite is more comprehensive than the equivalent CI test suite (e.g. due to `--fast-ci` flag for example), so there's value in a buildbot builder with "equivalent configuration". I also believe the release manager considers the buildbot statuses when making release decisions, not GitHub Actions.

I also noticed there are no refleak builders for MacOS at all (for the default build) -- would it be valuable to add such builders? (for default and/or nogil)

Reply:
> afaik the buildbot test suite is more comprehensive than the equivalent CI test suite

I see!

> I also noticed there are no refleak builders for MacOS at all (for the default build) -- would it be valuable to add such builders? (for default and/or nogil)

Oh, I didn't know that :( If possible for me, "yes"

Reply:
> If possible for me, "yes"

let me keep the existing builders as is (just marked as stable), and I'll look into adding refleak macos builders separately (probably marked "unstable" initially)

Reply:
> I also believe the release manager considers the buildbot statuses when making release decisions, not GitHub Actions.

Yes, it's here: https://peps.python.org/pep-0101/#how-to-make-a-release

> * Check the stable buildbots.
> 
>   Go to https://buildbot.python.org/all/#/release_status
>
>   Look at the buildbots for the release you’re making. Ignore any that are offline (or inform the community so they can be restarted). If what remains are (mostly) green buildbots, you’re good to go. If you have non-offline red buildbots, you may want to hold up the release until they are fixed. Review the problems and use your judgement, taking into account whether you are making an alpha, beta, or final release.

Although GitHub Actions is kind of implicitly included, because all PRs should be green before merging in the first place.